components/atom/tag/src/index.scss

```diff
@@ -83,4 +84,18 @@ $p-atom-tag: 0 $p-l !default;
       background-color: $c-primary-dark;
     }
   }
+
+  &-small {
+    height: $p-xl;
```

Comment:
I'm not sure if we should use here `$p-xl` when the default size is using a `$h-atom-tag`. So, while the default one is customizable via local variables to the component, the small size is customizable using global ones. We might want to use a multiplier here in order to create the small using `$h-atom-tag * 0.8` for example.

The same happen with the line-height and the padding.

Other solution could be having also component variables for the small size.

Reply:
ok! tx!